frontend_app_file/src/container/File.jsx

```diff
@@ -638,13 +638,17 @@ class File extends React.Component {
 
   getOnlineEditionAction = () => {
     const { state } = this
-    if (!appOfficeDocument) {
+    try {
+      if (!appOfficeDocument) {
+        return null
+      }
+      return appOfficeDocument.default.getOnlineEditionAction(
+        state.content,
+        state.config.system.config.collaborative_document_edition
+      )
+    } catch (error) {
```

Comment:
Don't use silent try catch.
Please add a console.error displaying the error and a comment to explain why you need a try/catch here